src/controls/qml/Particles.qml

```diff
@@ -0,0 +1,296 @@
+/*
```

Comment:
This file should be renamed to `Particle`. `Particles` is misleading as it really only represents a single particle.